mitmproxy/flow.py

```diff
@@ -169,6 +169,8 @@ def kill(self, master):
             self.reply.take()
         self.reply.kill(force=True)
         self.reply.commit()
+        self.live = False
+        master.addons("kill", self)
```

Comment:
I'm not against adding a kill event, but we need to do it properly, please. That means listing it in events.py, adding docs for it, and so forth.